src/box/sql.c

```diff
@@ -853,36 +853,10 @@ tarantoolsqlIdxKeyCompare(struct BtCursor *cursor,
 int
 tarantoolsqlIncrementMaxid(uint64_t *space_max_id)
```

Comment:
This change makes the function name and the op code obsolete - the function doesn't simply increment the max id anymore, instead it generates a space id. Also, this change turns this function into a one-liner so better just inline it into the op code handler. Anyway, this change should be reviewed by @ImeevMA.

Reply:
Dropped the function.